<ui version="4.0" >
 <class>ProcessOutputDialog_base</class>
 <widget class="QDialog" name="ProcessOutputDialog_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Process Output</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="labelCommandLine" >
     <property name="text" >
      <string>Command line:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="editCommandLine" >
     <property name="autoFormatting" >
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="lineWrapColumnOrWidth" >
      <number>428</number>
     </property>
     <property name="overwriteMode" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStdout" >
     <property name="text" >
      <string>Standard output:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="editStdout" >
     <property name="autoFormatting" >
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="lineWrapColumnOrWidth" >
      <number>428</number>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStderr" >
     <property name="text" >
      <string>Standard error:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="editStderr" >
     <property name="autoFormatting" >
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="lineWrapColumnOrWidth" >
      <number>428</number>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
